### DISCUSSION TITLE
Why does Copilot require 8 alphanumeric codes for login on Code-server

### DISCUSSION THREAD

**Discussion by heroyanglonghui** [coder/code-server#6254] [Q&A]
Hello
I am using **Code-serverV4.7.0**
manually installed the Copilot plug-in, and then login requires an **8 alphanumeric codes** . 
I asked the people on the Copilot team, and their answer was that this is the behavior of the IDE.
I also searched for related issues [#4998](https://github.com/coder/code-server/issues/4998#issuecomment-1077372928) and found that it is indeed Version 4.2.0 supports the use of 8 alphanumeric codes to log in to Copilot. 
I would like to know how to configure and turn off this step of verification?
<img width="1105" alt="image" src="https://github.com/coder/code-server/assets/20876976/b7367715-3487-4b30-a688-b40de893eee0">

@jsjoeio @code-asher

**Comment by code-asher** [comment]
Does it behave the same way in Codespaces or is this something unique to code-server?  If the former then we could try bringing the issue upstream (https://github.com/microsoft/vscode/).  I think this is the expected behavior from VS Code.
